src/cr_kyoushi/statemachines/aecid_attacker/config.py

```diff
@@ -143,6 +170,16 @@ class WordpressAttackConfig(BaseModel):
         description="The commands to execute via the web shell",
     )
 
+    wordpress_extra_args: List[str] = Field(
+        ["--disable-tls-checks", "--no-update"],
```

Comment:
Note that with no-update you have to ensure that "wpscan --update" is on install otherwise no vuln db will be present at all.

Reply:
I am not sure how to resolve this. As far as I know, these are the default options that were already there before I started templating the variables.

Reply:
See testbed gitlab

Reply:
My changes in gitlab need to be run and tested. But we can't run it now because it would overwrite this pull-request-changes in out test-instance. We will test the wpscan-update-things after this pull request was accepted